INN/umbrella-currentorg
# Update LTW shortcode media queries

## Changes

This pull request makes the following changes:

- Updates the min and max width media queries for the LTW shortcode stylings from 771px and 770px to 981px and 980px

Before on iPhone X:
![Screen Shot 2020-07-23 at 9 30 30 AM](https://user-images.githubusercontent.com/18353636/88292276-74e5d900-ccc7-11ea-98f5-824597860815.png)

After on iPhone X:
![Screen Shot 2020-07-23 at 9 30 09 AM](https://user-images.githubusercontent.com/18353636/88292281-76170600-ccc7-11ea-828a-d7909d2a27be.png)

## Why



For #162 

## Testing/Questions

Features that this PR affects:

- Mobile version of LTW shortcode


Questions that need to be answered before merging:

- [ ] Is this new target size for the media queries ok?

Steps to test this PR:

1. View a page with the LTW shortcode and make sure the mobile version works as expected
